### ISSUE TITLE
Version for Linux

### ISSUE THREAD

**Issue by DaniellMesquita** [ephtracy/ephtracy.github.io#196]

**Comment by irwant** [comment]
I hope this app available for linux. for now I open windows version using wine on ubuntu. its run smoothly

**Comment by IllusiveMan196** [comment]
Isn't it possible to make/compile it from source code provided in tar.gz archive?

**Comment by TheDrawingCoder-Gamer** [comment]
bumping this; if it can build for windows and mac I see no reason why it can't build for linux

**Comment by Tryzen-0** [comment]
yep... i wonder why no one ever tried to fork to get it running on linux natively.
I'm having graphical issues through wine
